apps/acqdat_api/lib/acqdat_api/dashboard_management/widget_instance.ex

```diff
@@ -297,7 +297,9 @@ defmodule AcqdatApi.DashboardManagement.WidgetInstance do
 
   def cal_yoffset(widget_layouts) do
     if widget_layouts != nil do
-      values = Map.values(widget_layouts)
+      values =
+        Map.values(widget_layouts)
+        |> Enum.reject(fn x -> is_nil(x) end)
 
```

Comment:
IMHO we should avoid single piping 
```suggestion
        widget_layouts
        |> Map.values()
        |> Enum.reject(fn x -> is_nil(x) end)

```

Reply:
fixed